server/package.json

```diff
@@ -19,6 +19,7 @@
     "@types/supertest": "^2.0.12",
     "@typescript-eslint/eslint-plugin": "^5.60.1",
     "@typescript-eslint/parser": "^5.60.1",
+    "axios-mock-adapter": "^1.21.5",
```

Comment:
Opinion of mine, but I don't think we need this package. I think we can just mock `axios` directly, especially since our use-case is very simple. What do you think @charliesantos and @kpchoy?

Reply:
It's for dev only, makes the test simpler, and not have to do extra work to create the mock, so I'm ok keeping it. We should remove it if we see issues. What are your concerns @mhuynh5757 ?

Reply:
It's for dev only, makes the test simpler, and not have to do extra work to create the mock, so I'm ok keeping it. We should remove it if we see issues. What are your concerns @mhuynh5757 ?

Reply:
I strongly dislike dependencies, and we're already mocking `axios` in `server/__mocks__`. I don't think it's more than 15 lines to mock `axios.get` ourselves, we just need to modify the `jest.fn` and give it a somewhat proper return value.

Reply:
Since we're already mocking axios, it's a good reason to not use a separate dependency for mocking. I'm ok removing it.
However, when bringing in dependencies in the future, we should evaluate the dependency objectively, weighing in pros and cons and let the data decide. Just saying we dislike dependencies in general does not help on the decision making process.

Reply:
Ok. Objectively speaking, the `axios-mock-adapter` dependency has few dependencies of its own, just two. And you're correct in that it's only a dev dependency. Furthermore, it seems quite popular, 1m+ DLs a week.

However, I see little benefit of using it since our use-case is so simple, and we only use `axios` in one place. If something breaks with the dependency, even though it's only a dev dependency means that our pipelines will break, our builds won't be green until we address it, etc. It's just more work for us.

Reply:
+1. I'm convinced.

Reply:
done